ioctl/newcmd/config/config.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package config
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+
+	serverCfg "github.com/iotexproject/iotex-core/config"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/pkg/log"
+)
+
+// Regexp patterns
+const (
+	_ipPattern               = `((25[0-5]|2[0-4]\d|[01]?\d\d?)\.){3}(25[0-5]|2[0-4]\d|[01]?\d\d?)`
+	_domainPattern           = `[a-zA-Z0-9][a-zA-Z0-9_-]{0,62}(\.[a-zA-Z0-9][a-zA-Z0-9_-]{0,62})*(\.[a-zA-Z][a-zA-Z0-9]{0,10}){1}`
+	_urlPattern              = `[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)`
+	_localPattern            = "localhost"
+	_endpointPattern         = "(" + _ipPattern + "|(" + _domainPattern + ")" + "|(" + _localPattern + "))" + `(:\d{1,5})?`
+	_defaultAnalyserEndpoint = "https://iotex-analyser-api-mainnet.chainanalytics.org"
+	_defaultConfigFileName   = "config.default"
+)
+
+var (
+	_supportedLanguage = []string{"English", "中文"}
+	_validArgs         = []string{"endpoint", "wallet", "explorer", "defaultacc", "language", "nsv2height"}
+	_validGetArgs      = []string{"endpoint", "wallet", "explorer", "defaultacc", "language", "nsv2height", "analyserEndpoint", "all"}
+	_validExpl         = []string{"iotexscan", "iotxplorer"}
+	_endpointCompile   = regexp.MustCompile("^" + _endpointPattern + "$")
+)
+
+// Info contains the information of config file
+type Info struct {
+	readConfig        config.Config
+	defaultConfigFile string
+}
+
+// InitConfig load config data from default config file
+func InitConfig() (config.Config, string) {
```

Comment:
return error

Reply:
no need to return err. because  if err!=nil, panic.

Reply:
1. Multiple places return error
2. the error handling logic is the same, `panic`

Thus, returning the error and handle it in the caller side is a better solution